spec/features/admin/api_users/create_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe 'Admin creates an API User', type: :feature, js: true do
+  let!(:admin) { create :admin }
+
+  let(:name) { Faker::Name.first_name }
+  let(:contact_email) { Faker::Internet.email }
+
+  before do
+    log_in admin
+    visit admin_api_users_path
+    click_link I18n.t('actions.add', model_name: ApiUser.model_name.human)
+  end
+
+  it 'creates API user' do
+    fill_in ApiUser.human_attribute_name('name'), with: name
```

Comment:
I saw that pattern in many specs in Murfin, but I'm still not convinced to that way of testing forms. Test should say verbosely what label exactly we expect imho.

Reply:
Yeah, I'm not convinced either tbh. I was following the convention that was already in existing specs.
I guess it's quite likely that on other installations the contents of the en_GB.yml would be changed and that would then break the tests. The question is would that matter, is anyone else likely to be running the tests or be contributors... I'm not sure?
I'll put it up as a new story on PT and we can discuss in tech review.

Reply:
good to know that I'm not alone with this preference :)